src/main/java/uk/ac/ebi/embl/api/validation/check/file/TSVFileValidationCheck.java

```diff
@@ -40,7 +40,6 @@
 	public final static String TEMPLATE_FILE_NAME = "TEMPLATE_";
 	private final static String TEMPLATE_ID_PATTERN = "(ERT[0-9]+)";
 	private final static String TEMPLATE_ACCESSION_LINE = "#template_accession";
-	private final static int MAX_SEQUENCE_COUNT = 100000;
 
```

Comment:
- We should also have the max sequence check for both TSV and flat files submitted as SEQUENCE_FLATFILE analysis type.